README.md

```diff
@@ -97,6 +97,7 @@ Feel free to add information about your own db-tools or your favorite third-part
 - [pspg](https://github.com/okbob/pspg) - Postgres Pager.
 - [SQLcl](http://www.oracle.com/technetwork/developer-tools/sqlcl/overview/index.html) - Oracle SQL Developer Command Line (SQLcl) is a free command line interface for Oracle Database.
 - [usql](https://github.com/xo/usql) - A universal command-line interface for PostgreSQL, MySQL, Oracle Database, SQLite3, Microsoft SQL Server, [and many other databases](https://github.com/xo/usql#database-support) including NoSQL and non-relational databases!
+- [Weaviate CLI](https://www.semi.technology/developers/weaviate/current/client-libraries/cli.html) - A CLI tool to interact with a [Weaviate](https://github.com/semi-technologies/weaviate) instance.
```

Comment:
This link https://www.semi.technology/developers/weaviate/current/client-libraries/cli.html
Should be this one:  https://github.com/semi-technologies/weaviate-cli